src/java/fr/paris/lutece/portal/service/filter/MainFilter.java

```diff
@@ -174,12 +174,11 @@ boolean matchFilterUrl( String strUrlPattern, String strRequestUrl )
 
         if ( strUrlPattern.endsWith( "/*" ) )
         {
-            if ( strUrlPattern.regionMatches( 0, strRequestUrl, 0, strUrlPattern.length( ) - 2 ) )
+            if ( strUrlPattern.regionMatches( 0, strRequestUrl, 0, strUrlPattern.length( ) - 2 )
+                    && strRequestUrl.length( ) == ( strUrlPattern.length( ) - 2 )
+                    || '/' == strRequestUrl.charAt( strUrlPattern.length( ) - 2 ) )

```

Comment:
Not sure that this is the same condition. Shouldn't the conditions of the second if be wrapped here in parenthesis ?
Is this case covered in `MainFilterTest` ?

Reply:
Will be corrected with missing TestCase